sections/main-product.liquid

```diff
@@ -106,6 +106,45 @@
                   {{ form | payment_terms }}
                 {%- endform -%}
               </div>
+            {%- when 'inventory' -%}
+              <p
+                class="product__inventory no-js-hidden{% if block.settings.text_style == 'uppercase' %} caption-with-letter-spacing{% elsif block.settings.text_style == 'subtitle' %} subtitle{% endif %}"
+                {{ block.shopify_attributes }}
+                id="Inventory-{{ section.id }}"
+                role="status"
+              >
+                {%- if product.selected_or_first_available_variant.inventory_quantity != 0 -%}
+                  {%- if product.selected_or_first_available_variant.inventory_quantity <= block.settings.inventory_threshold -%}
+                    <svg width="15" height="15" role="presentation">
+                      <circle cx="7.5" cy="7.5" r="7.5" fill="rgb(238,148,65, 0.3)"/>
+                      <circle cx="7.5" cy="7.5" r="5" stroke="rgb(255, 255, 255)" stroke-width="1" fill="rgb(238,148,65)"/>
+                    </svg>
+                    {%- if block.settings.show_inventory_quantity -%}
+                      {{- 'products.product.inventory_low_stock_show_count' | t: quantity: product.selected_or_first_available_variant.inventory_quantity -}}
+                    {%- else -%}
+                      {{- 'products.product.inventory_low_stock' | t -}}
+                    {%- endif -%}
+                  {%- else -%}
+                    <svg width="15" height="15" role="presentation">
+                      <circle cx="7.5" cy="7.5" r="7.5" fill="rgb(62,214,96, 0.3)"/>
+                      <circle cx="7.5" cy="7.5" r="5" stroke="rgb(255, 255, 255)" stroke-width="1" fill="rgb(62,214,96)"/>
+                    </svg>
+                    {%- if block.settings.show_inventory_quantity -%}
+                      {{- 'products.product.inventory_in_stock_show_count' | t: quantity: product.selected_or_first_available_variant.inventory_quantity -}}
+                    {%- else -%}
+                        {{- 'products.product.inventory_in_stock' | t -}}
+                    {%- endif -%}
+                  {%- endif -%}
+                {%- else -%}
+                  {%- if product.selected_or_first_available_variant.inventory_policy == 'continue' -%}
+                    <svg width="15" height="15" role="presentation">
+                      <circle cx="7.5" cy="7.5" r="7.5" fill="rgb(62,214,96, 0.3)"/>
+                      <circle cx="7.5" cy="7.5" r="5" stroke="rgb(255, 255, 255)" stroke-width="1" fill="rgb(62,214,96)"/>
+                    </svg>
+                    {{- 'products.product.inventory_in_stock' | t -}}
+                  {%- endif -%}
```

Comment:
Even though we already have a sold out badge, I'd prefer if we had a sold out/out of stock message rather than a blank. 
Ideally, i think the blank would only come up for unavailable variants: [video](https://screenshot.click/18-07-xdlac-4bivj.mp4)

cc: @YoannJailin

Reply:
Nice for designs 👌 And yeah I feel like it'd be nicer with it. If the block is supposed to show me the inventory status then show it to me in all scenarios, when it's out of stock, low stock and lots of stock. If I don't see it for an unavailable variant, I'm ok because it's expected as it doesn't exist.

Reply:
@YoannJailin it should be an easy thing. I'll update it soon.